src/js/themes/base.d.ts

```diff
@@ -0,0 +1,825 @@
+import {DeepReadonly} from '../utils'
+
+export declare const base: DeepReadonly<ThemeType>;
+export declare const generate: (baseSpacing = 24, scale = 6) => DeepReadonly<ThemeType>;
+
+declare const colors: {
+    active: string;
+    black: string;
+    border: {
+        dark: string;
+        light: string;
+    };
+    brand: string;
```

Comment:
I think all colors should be able to accept the ColorType from https://github.com/grommet/grommet/blob/8b25e3abd6fc42ec9977053c750ccb7854395d55/src/js/utils/index.d.ts#L43

In the sandbox example I created, I'm showing how brand color isn't necessarily a string, but can also accept an object.
https://codesandbox.io/s/colors-i19hp